src/Catel.Core/CoreModule.cs

```diff
@@ -49,7 +49,8 @@ public void Initialize(IServiceLocator serviceLocator)
             serviceLocator.RegisterType<IXmlSerializer, XmlSerializer>();
             serviceLocator.RegisterType<IXmlNamespaceManager, XmlNamespaceManager>();
             serviceLocator.RegisterType<ISerializationManager, SerializationManager>();
-            serviceLocator.RegisterType<IObjectAdapter, ObjectAdapter>();
+            serviceLocator.RegisterType<Catel.Runtime.Serialization.IObjectAdapter, Catel.Runtime.Serialization.ObjectAdapter>();
+            serviceLocator.RegisterType<Catel.Data.IObjectAdapter, Catel.Data.ReflectionObjectAdapter>();
```

Comment:
Was there a reason these changed to being reference via full namespace? It seems inconsistent (I also noticed it in several other files).